apps.toml

```diff
@@ -3465,6 +3465,14 @@ state = "working"
 subtags = [ "mediacenter" ]
 url = "https://github.com/YunoHost-Apps/streama_ynh"
 
+[streams]
+category = "social_media"
+level = 
+potential_alternative_to = [ "Facebook", "Twitter", "Tumblr" ]
```

Comment:
Align with #1704 and call Twitter X?

Reply:
Would make sense. Would github allow me to change the file while there's a PR?